### DISCUSSION TITLE
Snapshot testing

### DISCUSSION THREAD

**Discussion by MMDori** [KevinVandy/mantine-react-table#109] [Q&A]
Hello! I'm trying to write some tests for the table and when it comes to snapshot testing, it seems like some elements are always getting a new id. This causes the snapshots to fail all the time (example in the picture). Is there some sort of table props that can be used to control these id's?

I'm using  react-test-renderer: 18.2.0 ,  mantine-react-table: 1.0.0 and react: 18.2.0.

![image](https://github.com/KevinVandy/mantine-react-table/assets/142504606/224bb140-f7a2-4cc1-a469-323c4b5722b3)

**Comment by KevinVandy** [comment]
That might be a question for Mantine itself

**Comment by MMDori** [comment]
A colleague of mine found a solution. Maybe it will help others too. It appears there is a hook called 'useId' inside Mantine that generates the id's. Mocking it fixed the snapshot testing for me.

![image](https://github.com/KevinVandy/mantine-react-table/assets/142504606/4371e9cf-624c-458d-a8ab-fd5a3be29345)
